instrumentation/aws-java-sdk-s3-2.0/src/test/java/com/agent/instrumentation/awsjavasdks3/AmazonS3AsyncApiTest.java

```diff
@@ -103,35 +103,35 @@ public void testListBuckets() {
 //        createBucketNoTxn();
 //        putObjectNoTxn();
 //        getObject();
-//        assertMetrics("getObject");
+//        assertMetrics("getObject", null);
 //    }
 
     @Test
     public void testListObjects() {
         createBucketNoTxn();
         listObjects();
-        assertMetrics("listObjects");
+        assertMetrics("listObjects", 200);
     }
 
     @Test
     public void testPutObject() {
         createBucketNoTxn();
         putObject();
-        assertMetrics("putObject");
+        assertMetrics("putObject", 200);
     }
 
     @Test
     public void testDeleteBucket() {
         createBucketNoTxn();
         deleteBucket();
-        assertMetrics("deleteBucket");
+        assertMetrics("deleteBucket", 204);
     }
 
     @Test
     public void testGetBucketLocation() {
         createBucketNoTxn();
         getBucketLocation();
-        assertMetrics("getBucketLocation");
+        assertMetrics("getBucketLocation", 200);
     }
 
 // This test passes but it consistently takes 60s to complete
```

Comment:
So do we ever uncomment and run this test or should it go?

Reply:
We can uncomment it now that we have GE.